src/applications/disability-benefits/all-claims/tests/schema/maximal-test.json

```diff
@@ -349,7 +349,8 @@
         "from": "1992-02-16",
         "to": "1995-04-04"
       },
-      "incidentDate": "1994-04-03"
+      "incidentDate": "1994-04-03",
+      "view:uploadSources": true
```

Comment:
Added `"view:uploadSources": true` to be able to access the supporting evidence document upload data for 781a